docs/usage/usage.md

```diff
@@ -81,6 +80,25 @@ autoScaling:
   mode: "vpa"
 ```
 
+An example `NetworkingConfig` manifest for static resource assignment:
+
+```yaml
+apiVersion: calico.networking.extensions.gardener.cloud/v1alpha1
+kind: NetworkConfig
+autoScaling:
+  mode: "static"
+  resources:
+    node:
+      cpu: 100m
+      memory: 100Mi
+    typha:
+      cpu: 100m
+      memory: 100Mi
+```
+
+> ℹ️ Please note that it is only possible to (optionally) configure the resource requests for `calico-node` and `calico-typha` in static mode.
```

Comment:
I'm not sure if this sentence is really understandable. At least, I got it wrong reading it for the first time.
It is not needed, to configure both `calico-node` **and** `calico-typha` and the `optionally` means that if resources are not specified there is a default.
Maybe, we should use the default values in this example.
suggestion:
```
"Please note that in static mode, you have the option to configure the resource requests for calico-node and calico-typha. If not specified, default settings will be used."
```

Reply:
Adapted the section to hopefully be more clear.
WDYT?